### PR TITLE
Bring truffle code up to date with upstream repo

### PR DESCRIPTION
**Pull request by snim2**

### CODE REVIEW

File: lib/eco/export/jruby_simple_language.py [hunk 1]
```diff
@@ -92,24 +92,18 @@ def _walk_rb(self, node):
...
 
-    def _apply_template(self, name, magic_name):
-        return """  {1} = Truffle::Interop.import('{0}')
-  def {0}(*args)
-    Truffle::Interop.execute({1}, *args)
-  end
-
-""".format(name, magic_name)
+    def _apply_template(self, name):
+        return "Truffle::Interop.import_method(:{0})".format(name)
```
Comment (ltratt) [inline lib/eco/export/jruby_simple_language.py]: Is there any reason for using `format` over `%`?

Comment (snim2) [inline lib/eco/export/jruby_simple_language.py]: Nothing in particular

Comment (ltratt) [inline lib/eco/export/jruby_simple_language.py]: Given that `%` is more common, should we use that then?

**Comment by ltratt** [comment]
Do you want to squash, then we can merge?

**Comment by snim2** [comment]
OK, all squashed...

**Comment by ltratt** [comment]
Thanks!

**Comment by ptersilie** [comment]
Even though I prefer the `%` myself, it seems `format` is the way to go: https://docs.python.org/release/3.1.5/library/stdtypes.html#old-string-formatting-operations

> Note: The formatting operations described here are obsolete and may go away in future versions of Python. Use the new String Formatting in new code.

**Comment by ltratt** [comment]
I don't mind too much, but I think we should be consistent within this codebase. Currently we use `%`. We could easily replace all the `%`s with `format` in a PR, and then we could move over to `format` permanently. [I'm not going to raise such a PR, or be hugely interested in reviewing it, but I wouldn't object to it either.]

**Comment by snim2** [comment]
OK, I will add this as an issue so we don't forget, but I guess it won't be a priority any time soon.

**Comment by ptersilie** [comment]
My point wasn't that we should switch over to `format`. I just wanted to point out that it's ridiculous that we are even discussing whether we should use one or the other, because it is completely irrelevant and not worth our time.

**Comment by ltratt** [comment]
Code consistency within a codebase isn't irrelevant -- the less consistent we are, the harder it is to maintain things over time.

**Pull request merged**